DynamicLights/Izhikevich.cpp

```diff
@@ -0,0 +1,195 @@
+//
+//  Izhikevich.cpp
+//
+//
+// Copyright 2020, Atsushi Masumori, Alexandre Saunier, Simon Demeule
```

Comment:
I think I wouldn't add more copyright owners, here - it simply makes relicensing it harder. I would perhaps add Xmodal, but not individuals - or event better: add no one.

Reply:
So would «Xmodal & Atsushi Masumori» be our best pick for credits? Also, should I just add this header everywhere?